src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Options/OptionsStorage.cs

```diff
@@ -64,6 +66,12 @@ public SnippetSetting Snippets
         set => SetInt(SnippetsName, (int)value);
     }
 
+    public LogLevel LogLevel
+    {
+        get => (LogLevel)GetInt(LogLevelName, (int)LogLevel.Warning);
```

Comment:
I just merged the options changes :/

Reply:
Yeah, I know :(

Also, my option doesn't appear in the unified settings screen. Is there something else I need to do?

Reply:
It looks right to me. The only thing I can think of is some weird caching